cmake/EthDependencies.cmake

```diff
@@ -43,62 +43,9 @@ find_program(CTEST_COMMAND ctest)
 
 ## use multithreaded boost libraries, with -mt suffix
 set(Boost_USE_MULTITHREADED ON)
+set(Boost_USE_STATIC_LIBS ON)
```

Comment:
Please only measure the release binary. If the release binary is 66M, we are in serious trouble.